packages/styled-ui-docs/pages/drawer.mdx

```diff
@@ -166,70 +363,99 @@ function Example() {
 }
 ```
 
-### Placement
+### Transition
 
-Pass the `placement` prop if you need to adjust the placement of the Drawer. Values can be
- `right` or `left`. Default is `right`.
+The drawer doesn't come with a transition by default. You can use transition components to provide simple transitions.
 
-```jsx
-function SizeExample() {
-  const { isOpen, onOpen, onClose } = useDisclosure();
-  const [placement, setPlacement] = React.useState('right');
+#### Adding slide transition
 
-  const handlePlacementClick = newPlacement => {
-    setPlacement(newPlacement);
-    onOpen();
-  };
+```
+import { Slide } from '@trendmicro/react-styled-ui';
+```
 
-  const placements = ['left', 'right'];
+```jsx
+function Example() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
 
   return (
     <>
-      {placements.map(placement => (
-        <Button
-          onClick={() => handlePlacementClick(placement)}
-          key={placement}
-          m={4}
-        >{`Open ${placement} Drawer`}</Button>
-      ))}
-
-      <Drawer onClose={onClose} placement={placement} isOpen={isOpen}>
-        <DrawerOverlay />
-        <DrawerContent>
-          <DrawerHeader>Drawer Title</DrawerHeader>
-          <DrawerBody>
-            Bla bla bla...
-            Bla bla bla...
-            Bla bla bla...
-          </DrawerBody>
-          <DrawerFooter>
-            <Button onClick={onClose}>Close</Button>
-          </DrawerFooter>
-        </DrawerContent>
-      </Drawer>
+      <Button onClick={onOpen}>
+        Launch drawer
+      </Button>
+      <Slide
+        in={isOpen}
+        from="right"
+        finalHeight="100vh"
+      >
+        {styles => (
+          <Drawer
+            autoFocus={false}
+            backdrop
+            isCloseButtonVisible
+            isOpen={isOpen}
+            closeOnEsc
+            closeOnOutsideClick
+            onClose={onClose}
+            placement="right"
+          >
+            <DrawerOverlay />
```

Comment:
```jsx
<DrawerOverlay opacity={styles.opacity} />
```

Reply:
Fixed in bd9d1faa84f2591d14982c6f3875cbacf1740a75 and 7c9fb92f249cce05ba7d239c6ffaa7837408d158